sql/src/test/java/org/apache/druid/sql/calcite/CalciteInsertDmlTest.java

```diff
@@ -743,6 +743,38 @@ public void testInsertWithInvalidSelectStatement()
         .verify();
   }
 
+  @Test
```

Comment:
Should we also add a test that tries to ingest a forbidden column name (like EXPR$0)? This would also help in documenting the behavior in the tests, and if we get a workaround to that method, that test would fail and would need to be updated.

Reply:
Added a test for this case